src/utils/wrapData.ts

```diff
@@ -0,0 +1,63 @@
+export enum IndexTokenMix {
```

Comment:
@ElmoArmy should we call this `Index`TokenMix or `Set`TokenMix

Maybe a general question if prefer using `index` now or still `set` for naming.

Reply:
Generally, I like the idea of naming things "Index" over "Set" when interfacing with our forked protocol.